### ISSUE TITLE
Output of pip is interpreted as a format string

### ISSUE THREAD

**Issue by tgs** [mitsuhiko/platter#16]
For example, pip has a warning `DEPRECATION: Failed to find 'wraputil' at (URL). It is suggested to upgrade your index to support normalized names as the name in /simple/{name}.`  But `process_stream_output` sends that as the first argument to `info`, and `info` in turn looks for a `name` kwarg, and doesn't find it.

```
[...]
  File "/home/smithtg/.local/venvs/platter/lib/python2.7/site-packages/platter.py", line 549, in build
    self.build_wheels(venv_path, data_dir)
  File "/home/smithtg/.local/venvs/platter/lib/python2.7/site-packages/platter.py", line 329, in build_wheels
    self.execute(os.path.join(venv_path, 'bin', 'pip'), cmdline)
  File "/home/smithtg/.local/venvs/platter/lib/python2.7/site-packages/platter.py", line 262, in execute
    self.log.process_stream_output(cl)
  File "/home/smithtg/.local/venvs/platter/lib/python2.7/site-packages/platter.py", line 139, in process_stream_output
    self.info(click.style(line.rstrip(), fg=color))
  File "/home/smithtg/.local/venvs/platter/lib/python2.7/site-packages/platter.py", line 119, in info
    click.echo(prefix + fmt.format(*args, **kwargs))
KeyError: 'name'
```

**Comment by tgs** [comment]
Changing line 139 to this fixes it:
    self.info('{}', click.style(line.rstrip(), fg=color))

But I haven't checked for more instances of the same pattern.

**Comment by msabramo** [comment]
I'm running into the same issue.

@tgs: Thanks for posting the issue and a possible solution.

**Comment by msabramo** [comment]
See #25

**Comment by RR2DO2** [comment]
This was resolved by #31
